sbin/fetch_micro_bench_artifact.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
This can probably be significantly simplified by using GitHub's CLI (`gh`) rather than going through the raw API.